### ISSUE TITLE
socklog does not work out of the box with read-only rootfs

### ISSUE THREAD

**Issue by mikroskeem** [just-containers/socklog-overlay#6]
When using docker's `--read-only` and s6-overlay's `S6_READ_ONLY_ROOT=1`, then /var/log directory needs to be created before socklog can be used. Otherwise socklog yields error similar to #2 nonstop.

Considering that this is an add-on to s6-overlay, then perhaps this overlay could install an additional script setting up /var/log directory in case of `S6_READ_ONLY_ROOT` is enabled?

```sh
cat > /etc/cont-init.d/00-init-varlog <<EOF
#!/usr/bin/execlineb -P
mkdir -p /var/log
EOF
```

**Comment by jprjr** [comment]
There's already a cont-init.d script in this repo that's calling
mkdir -p for /var/log/socklog/etc...

Are you running with a writable volume at /var?

**Comment by mikroskeem** [comment]
Yes, /var is writable (it's tmpfs, I followed https://github.com/just-containers/s6-overlay#read-only-root-filesystem)
